config/kernel.m4

```diff
@@ -601,9 +601,18 @@ dnl #
 dnl # Used internally by ZFS_LINUX_TEST_{COMPILE,MODPOST}
 dnl #
 AC_DEFUN([ZFS_LINUX_COMPILE], [
+	AC_ARG_VAR([KERNEL_CC], [C compiler for
+		building kernel modules])
+	AC_SUBST(KERNEL_CC)
```

Comment:
The last nit: AC_ARG_VAR already invokes AC_SUBST. No need to repeat.
https://www.gnu.org/software/autoconf/manual/autoconf-2.69/html_node/Setting-Output-Variables.html

Reply:
Thanks.
Forgot that.